systemverilog-plugin/UhdmAst.cc

```diff
@@ -4533,6 +4533,10 @@ AST::AstNode *UhdmAst::process_object(vpiHandle obj_handle)
     case vpiClockingBlock:
         process_unsupported_stmt(object);
         break;
+    case vpiTypeParameter:
+        // Surelog replaces parameters with specific values
+        //    rely on that, skip processing for now
+        break;
```

Comment:
Please add to the comment something like:
```
Instances in an `uhdmTopModules` tree already have all parameter references substituted with the parameter type/value by Surelog, so the plugin doesn't need to process the parameter itself.
Other parameter types are handled by the plugin mainly because they were implemented before Surelog did the substitution.
```